web/src/axios-orders.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const instance = axios.create({
+    baseURL: 'https://referral-3ffeb.firebaseio.com/'
```

Comment:
Do you get the form data at this endpoint?

Reply:
Yes. But, I used Firebase. The current baseURL link will be replaced by Postman Link.